src/themes/modern/main/ts/modes/Inline.ts

```diff
@@ -121,8 +121,6 @@ const render = function (editor, theme, args) {
     editor.on('nodeChange', reposition);
     editor.on('ResizeWindow', reposition);
     editor.on('activate', show);
-    editor.on('deactivate', hide);
-
```

Comment:
@vdtn359 I'd probably have suggested leaving this blank line in place for readability's sake